responses/pt-br/HassGetState.yaml

```
@@ -1,4 +1,126 @@
+        {% endif %}
+
+      how_many: |
+        {{ query.matched | length }}
+
+      where: |
+        {{ slots.name | capitalize }} está
+        {% if state.attributes.get("untranslated_state") == "not_home" %}
+          {{ state.state }}
+        {% else %}
+          em {{ state.state }}
+        {% endif %}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)